spacy_llm/tests/tasks/test_ner.py

```diff
@@ -221,52 +220,27 @@ def test_ner_config(config: Config):
 @pytest.mark.skipif(has_openai_key is False, reason="OpenAI API key not available")
 @pytest.mark.parametrize(
     "cfg_str",
-    ["fewshot_cfg_string_v3_lds", "fewshot_cfg_string_v3", "ext_template_cfg_string"],
+    ["fewshot_cfg_string_v3_lds", "fewshot_cfg_string_v3"],
 )
 @pytest.mark.parametrize(
     "text,gold_ents",
     [
-        # simple
         (
             "Marc and Bob both live in Ireland.",
             [("Marc", "PER"), ("Bob", "PER"), ("Ireland", "LOC")],
         ),
-        # same entity names
-        (
-            "Paris, who is a person, travelled to Paris, which is a city, to spend time with her friend Paris, who "
-            "is a person.",
-            [("Paris", "PER"), ("Paris", "LOC"), ("Paris", "PER")],
-        ),
```

Comment:
I'm removing it here because getting the LLM to predict it correctly, is too uncertain. We already have the test `test_regression_span_task_response_parse` which checks that the parsing works, given the right answer.